app/src/pages/inside/common/btsIntegrationSelector/btsIntegrationSelector.jsx

```diff
@@ -51,7 +50,7 @@ export class BtsIntegrationSelector extends Component {
     super(props);
     this.pluginNamesOptions = Object.keys(props.namedBtsIntegrations).map((key) => ({
       value: key,
-      label: PLUGIN_NAME_TITLES[key] || key,
+      label: key,
```

Comment:
With this change:
![image](https://github.com/user-attachments/assets/fd792077-a339-4a30-b2c1-41f50ecbf14d)
Without:
![image](https://github.com/user-attachments/assets/600f4eb9-ffa5-40b6-97b5-785c09c52bd2)

Plugin name should be used here as a first possible value and a `key` as a fallback.
So, each BTS plugin should also has `details.name` to be used here (applicable for Jira Server, Rally, Monday, Jira Cloud, Azure DevOps, GitLab).

Reply:
Feel free add it.

Reply:
Moreover, old plugin without name updates will be displayed like this.